src/fswAlgorithms/attGuidance/sunSearch/sunSearch.h

```diff
@@ -65,6 +65,7 @@ class SunSearch: public SysModel {
     BSKLogger bskLogger;            //!< BSK Logging
 
     void ComputeKinematicProperties(int const index);
+    void ComputeReferenceMotion(uint64_t const CurrentSimNanos, int const index, double *omega_RN, double *domega_RN);
```

Comment:
:large_orange_diamond: `CurrentSimNanos` can be `currentSimNanos`

Reply:
Done